.github/workflows/codeql.yml

```diff
@@ -0,0 +1,41 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+  schedule:
+    - cron: "9 15 * * 6"
```

Comment:
do we really need this?

Reply:
aren't we already checking on pushes to master? is this to refresh in case some checks got updated behind the scenes?

Reply:
I think it's overkill because it'll be checking more than just pushes. I think we can get rid of this.